src/app/components/workbox-tab.component.ts/workbox-tab.component.ts

```diff
@@ -0,0 +1,12 @@
+import {Component} from "@angular/core";
```

Comment:
Is this just Github, or is this in a directory called "workbox-tab.component.ts" ?